src/styles/_unused.less

```diff
@@ -0,0 +1,65 @@
+.ironsworn {
```

Comment:
Is this file meant to be deleted when you're sure you don't need it anymore?

Reply:
yeah, that's the idea. this PR is pretty firmly in the Draft Zone still, and i was mostly looking for feedback on its general direction before i sank more time into it.